docs/resources/tunnel_config.md

```diff
@@ -14,15 +14,15 @@ Provides a Cloudflare Tunnel configuration resource.
 ## Example Usage
 
 ```terraform
-resource "cloudflare_argo_tunnel" "example_tunnel" {
+resource "cloudflare_tunnel" "example_tunnel" {
```

Comment:
https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/examples/resources/cloudflare_tunnel_config/resource.tf#L1 is the file you want here